src/System.Net.Http/tests/UnitTests/project.json

```diff
@@ -12,12 +12,16 @@
     "System.Reflection.TypeExtensions": "4.1.1-beta-24403-05",
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Resources.ResourceManager": "4.0.2-beta-24403-05",
+    "System.Net.Http": {
```

Comment:
>I think it's OK to have the exclusion here: the test is including the appropriate production files.

Correct.  The UnitTests bring in the product code directly as part of compiling the test assemblies themselves.  So, you should not and need not include a reference to the System.Net.Http package itself.